gen-docs/README.md

```
@@ -25,3 +25,19 @@ $ pnpm build
 ```
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Test API Rest documentation
+
+This command generate API Rest documentation
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)